massa-execution-worker/src/execution.rs

```diff
@@ -858,6 +859,21 @@ impl ExecutionState {
             _ => panic!("unexpected operation type"),
         };
 
+        // Verify limits
+        if target_func.len() > self.config.max_function_length as usize {
```

Comment:
For callSC I think that we check this at deserialization (?)

Reply:
it's on SC-to-SC calls that we don't check I think

Reply:
Yes but I understood you asked here in the previous description of the issue, maybe I didn't understand well but you changed it so I will change it also.

Reply:
yeah I might have made a mistake :) here I made it clearer that it is SC-to-SC